sys/include/net/sock/dtls.h

```diff
@@ -0,0 +1,562 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
```

Comment:
BTW, just noticed this: I think the authors of the APIs this API is based on  (@danielinux, @rfuentess, @kb2ma) should also keep their copyright to this work. Similar as we included all people planning on the original `sock` into the copyright.

Reply:
Added the copyright and authors. I used names and emails from existing docs I found in RIOT. Not sure that is up to date though.